tests/gem/test_packets.py

```diff
@@ -51,6 +51,97 @@ def test_packet_with_extra_after(self):
         assert_packet("BIN32-NET.bin", packet)
 
 
+class TestPacketDeltaComputation(unittest.TestCase):
+    def test_packet_delta_seconds(self):
+        packet = parse_packet("BIN32-ABS.bin", packets.BIN32_ABS)
+        self.assertEqual(997492, packet.seconds)
+
+        self.assertEqual(997493, packet.delta_seconds(2 ** 24 - 1))
+        self.assertEqual(1000000, packet.delta_seconds(2 ** 24 - (1000000 - 997492)))
+
+    def test_packet_delta_pulses(self):
+        packet = parse_packet("BIN48-NET-TIME_tricky.bin", packets.BIN48_NET_TIME)
+
+        # All the pulse counts in our packets are 0, so let's fake some out
```

Comment:
Should we make a non-zero packet here instead of doing this in python?

I actually don't know how to construct one of these files, actually, so that's something we should probably document at some point.